<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="splitPage">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="setting_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0,0">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ip_lineEdit">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	padding: 4px 4px;
}</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="change_pushButton">
            <property name="text">
             <string>변경</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_pushButton">
            <property name="text">
             <string>삭제</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="load_pushButton">
            <property name="text">
             <string>불러오기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zoom_out_pushButton">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zoom_in_pushButton">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="videoPage">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <item>
        <layout class="QHBoxLayout" name="layout"/>
       </item>
       <item>
        <widget class="QWidget" name="setting_widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="back_pushButton">
            <property name="text">
             <string>뒤로가기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="play_pushButton">
            <property name="text">
             <string>재생</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pause_pushButton">
            <property name="text">
             <string>일시정지</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop_pushButton">
            <property name="text">
             <string>정지</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rewind_pushButton">
            <property name="text">
             <string>되감기</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forward_pushButton">
            <property name="text">
             <string>빨리감기</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
